main.go

```diff
@@ -165,11 +167,12 @@ func main() {
 		DefaultWorkloadClusterNamespaces: defaultNamespaces,
 	}
 
-	grafanaAgentSecret := grafanaagentsecret.Reconciler{
-		Client: mgr.GetClient(),
+	eventsLoggerConfig := k8seventsconfig.Reconciler{
```

Comment:
Why k8seventsconfig and eventsloggersecret? Both should have similar names right?

Reply:
You could maybe move them under `resource/event-logging/config|secret`

Reply:
> Why k8seventsconfig and eventsloggersecret? Both should have similar names right?

I agree, it's just because I began calling it `k8s-events` before switching to `events-logger` latter but I forgot to rename the package.

> You could maybe move them under resource/event-logging/config|secret

I would rather not to stay compliant with the other resources which have the secret separated from the logging-agent.

Reply:
Do you want me to move the logging-agent resources to another folder in another PR? :D

Reply:
Btw I'm sorry about the comments but this PR adds so much stuff that I sadly can't really review it as I would. And I'm not saying it's your fault :)